sections/03_categorize.md

```diff
@@ -28,7 +28,76 @@ we share these data while respecting privacy? I've made a section for each of
 these. Feel free to add more. I see each section as something on the order of
 1-2 paragraphs in our context.*
 
-#### Standardization/integration
+#### Clinical Care
+
+#### Imaging Modalities (static + dynamic)
+
+* Imaging #163, #164
+
+#### Electronic Health Records
+
+EHR data include substantial amounts of free text, which remains challenging to
+approach [@doi:10.1136/amiajnl-2011-000501]. Often, researchers developing
+algorithms that perform well on specific tasks must design and implement
+domain-specific features [@doi:10.1136/amiajnl-2011-000150]. These features
+capture unique aspects of the literature being processed. Deep learning methods
+are natural feature constructors. In recent work, the authors evaluated the
+extent to which deep learning methods could be applied on top of generic
+features for domain-specific concept extraction [@arxiv:1611.08373]. They found
+that performance was in line with, but did not exceed, existing state of the art
+methods. The deep learning method had performance lower than the best performing
+domain-specific method in their evaluation[@arxiv:1611.08373]. This highlights
+the challenge of predicting the eventual impact of deep learning on the field.
+This provides support that deep learning may impact the field by reducing the
+researcher time and cost required to develop specific solutions, but it may not
+lead to performance increases.
+
+TODO: survival analysis/readmission prediction methods from EHR/EMR style data
```

Comment:
Nevertheless, recent work has revealed one domain in which deep networks have proven superior to traditional methods. Survival analysis models the time leading to an event of interest from a shared starting point, and in the context of EHR data, often associates these events to subject covariates. Exploring this relationship is difficult, however, given that EHR data types are often heterogeneous, covariates are often missing, and conventional approaches require the covariate-event relationship be linear and aligned to a specific starting point (@arxiv:1608.02158). Early approaches, such as the Faraggi-Simon feed-forward network, aimed to relax the linearity assumption, but performance gains were lacking (@doi:10.1016/S0167-9473(99)00098-5). Katzman et al. in turn developed a deep implementation of the Faraggi-Simon network that, in addition to outperforming Cox regression, was capable of comparing the risk between a given pair of treatments, thus potentially acting as recommender system (@arxiv:1606.00931). To overcome the remaining difficulties, researchers have turned to deep exponential families, a class of latent generative models that are constructed from any type of exponential family distributions (@arxiv:1411.2581v1). The result was a deep survival analysis model capable of overcoming challenges posed by missing data and heterogeneous data types, while uncovering nonlinear relationships between covariates and failure time. They showed their model more accurately stratified patients as a function of disease risk score compared the current clinical implementation.